app/assets/tailwind/sure-design-system.css

```
@@ -0,0 +1,12 @@
+/*
+ * Sure design system entry.
+ * Tokens (theme, dark overrides, utilities) are generated from design/tokens/sure.tokens.json: see _generated.css.
+ * Element resets, components, and prose overrides remain hand-written.
+ */
+
+@custom-variant theme-dark (&:where([data-theme=dark], [data-theme=dark] *));
+
+@import './sure-design-system/_generated.css';
```

Stylelint (17.9.0)

[error] Invalid position for `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)